client/components/common/BaseForm.vue

```diff
@@ -108,4 +113,41 @@ span {
   display: flex;
   align-items: center;
 }
+
+.tooltip {
+  position: relative;
+  display: inline-block;
+}
+
+.tooltiptext {
+  visibility: hidden;
+  background-color: var(--white);
+  color: #fff;
+  text-align: center;
+  padding: 6px;
+  border-radius: 6px;
+
+  position: absolute;
+  z-index: 1;
+  width: max-content;
```

Comment:
Nitpicky, but I wonder if setting width to `max-content` is the really best thing to do here, since text can overflow if instructions are too long, e.g.: 
![image](https://user-images.githubusercontent.com/59969293/206897570-3668607a-e210-470c-b1c8-c58eeaacb800.png)

Reply:
I think because instructions are not a user input issue (but rather more on our end) this shouldn't be a problem – and if instructions are that long, they're probably not good instructions. I'd rather not hardcode the width in this case